packages/backend/src/config/app.ts

```diff
@@ -123,17 +126,38 @@ function parseRedisTlsConfig(
 
   // self-signed certs.
   if (caFile !== '') {
-    options.ca = readFileSync(caFile)
+    options.ca = fs.readFileSync(caFile)
     options.rejectUnauthorized = false
   }
 
   if (certFile !== '') {
-    options.cert = readFileSync(certFile)
+    options.cert = fs.readFileSync(certFile)
   }
 
   if (keyFile !== '') {
-    options.key = readFileSync(keyFile)
+    options.key = fs.readFileSync(keyFile)
   }
 
   return Object.keys(options).length > 0 ? options : undefined
 }
+
+function parseOrProvisionKey(keyPath: string): crypto.KeyObject {
+  try {
+    const key = crypto.createPrivateKey(fs.readFileSync(keyPath))
+    const jwk = key.export({ format: 'jwk' })
+    if (jwk.crv === 'Ed25519') {
+      return key
+    } else {
+      console.log('Private key is not EdDSA-Ed25519 key. Generating new key.')
+    }
+  } catch {
+    console.log('Private key could not be loaded. Generating new key.')
+  }
+  const keypair = crypto.generateKeyPairSync('ed25519')
+  fs.mkdirSync('../../tmp')
+  fs.writeFileSync(
+    '../../tmp/private.pem',
```

Comment:
Maybe make a const string used both here and in the `privateKey` assignment above

Reply:
Which `privateKey` assignment?

Reply:
https://github.com/interledger/rafiki/pull/638/files#diff-6300c4703a2fb930745656aed4f5d68507eb07a20ae1f817a1ca3c9d90901facR112

Reply:
maybe something like
```
const TMP_DIR = './tmp'
const PRIVATE_KEY_FILE = `${TMP_DIR}/private-key.pem`

Reply:
Oh, that far above 😄